www/front_src/src/Resources/Details/tabs/Details/DetailsCard/DetailsLine.tsx

```diff
@@ -8,7 +8,6 @@ import makeStyles from '@mui/styles/makeStyles';
 const useStyles = makeStyles((theme) => ({
   lineText: {
     fontSize: theme.typography.body2.fontSize,
-    overflow: 'hidden',
```

Comment:
Did you check that this change doesn't impact negatively whe the line is bigger than the tile?